src/common/file.rs

```diff
@@ -61,19 +54,11 @@ mod tests {
     fn test_file() {
         let content = b"Some Text";
         let file_name = "sample.parquet";
-        let resp = put_file_contents(file_name, content);
-        assert!(resp.is_ok());
-
-        let resp = get_file_contents(file_name);
-        assert_eq!(resp.unwrap(), content);
-
-        let resp = get_file_meta(file_name);
-        assert_eq!(resp.unwrap().is_file(), true);
-
-        let resp = scan_files(file_name);
-        assert!(resp.len() > 0);
 
-        let resp = delete_file(file_name);
-        assert!(resp.is_ok());
+        put_file_contents(file_name, content).unwrap();
+        assert_eq!(get_file_contents(file_name).unwrap(), content);
+        assert!(get_file_meta(file_name).unwrap().is_file());
+        assert!(!scan_files(file_name).is_empty());
+        std::fs::remove_file(file_name).unwrap();
```

Comment:
And test an std library.

Reply:
Why would we want to test standard library calls? :)

If they did't work, broken unit test would the least of our problems. If we suspected that Rust's standard library is insufficiently tested, we should have switched to different programming language immediately.

Reply:
i mean why call a std lib here, not let you add a test for std lib.